public/index.html

```diff
@@ -24,25 +18,26 @@ <h3 id="status">Disconnected</h3>
       <p id="final"></p>
     </div>
 
-    <div id="transcript">
-      <h3>Transcript</h3>
-      <p id="transcript-words"></p>
+ 
 
-      <h3>Words</h3>
-      <ul id="transcript-list"></ol>
-    </div>
-
-    <div id="entities">
-      <h3>Entities</h3>
-      <ul id="entities-list"></ol>
-    </div>
+    <div id="results" style="width:100%">
+      <div id="transcript" style="width: 33%; float:left" >
```

Comment:
@ottomatias have you tested this layout on a mobile device? I know that current layout isn't exactly mobile-friendly, but it's at least usable.

Reply:
It is tested and there should be enough room on mobile for the intents, entities and words but the log is hardly readable. 
<img width="442" alt="Näyttökuva 2020-5-13 kello 10 26 31" src="https://user-images.githubusercontent.com/2462500/81783577-3a600300-9504-11ea-86d7-b92d1086c069.png">

Reply:
Thanks for verifying! The log being unreadable is something we can live with for now, IMO.

Reply:
And one more thing: the reasoning for adding these columns is that for the screenshot the current layout doesn't show that much information as the important parts of the page kind of are in the bottom. 
I'm not sure how important that is, so again: your call on what we want to do.